open-metadata-implementation/repository-services/repository-services-implementation/src/main/java/org/odpi/openmetadata/repositoryservices/metadatahighway/OMRSMetadataHighwayManager.java

```diff
@@ -182,23 +182,26 @@ public CohortConnectionStatus connectToCohort(CohortConfig         cohortConfig)
                     = new OMRSRepositoryEventExchangeRule(cohortConfig.getEventsToProcessRule(),
                                                           cohortConfig.getSelectedTypesToProcess());
 
-            cohortManager.initialize(cohortConfig.getCohortName(),
-                                     localMetadataCollectionId,
-                                     localMetadataCollectionName,
-                                     localServerName,
-                                     localServerType,
-                                     localOrganizationName,
-                                     localRepository,
-                                     localRepositoryContentManager,
-                                     enterpriseAccessConnectionConsumer,
-                                     enterpriseAccessTopicConnector,
-                                     cohortRegistryStore,
-                                     cohortConfig.getCohortOMRSTopicConnection(),
-                                     cohortTopicConnector,
-                                     inboundEventExchangeRule);
+            OMRSCohortManagerInitializer initializer = new OMRSCohortManagerInitializer(cohortManager, 
+                    cohortConfig.getCohortName(),
+                    localMetadataCollectionId,
+                    localMetadataCollectionName,
+                    localServerName,
+                    localServerType,
+                    localOrganizationName,
+                    localRepository,
+                    localRepositoryContentManager,
+                    enterpriseAccessConnectionConsumer,
+                    enterpriseAccessTopicConnector,
+                    cohortRegistryStore,
+                    cohortConfig.getCohortOMRSTopicConnection(),
+                    cohortTopicConnector,
+                    inboundEventExchangeRule);
+            
+            initializer.start();
 
             /*
-             * The cohort manager is only added to the list if it initializes successfully.
+             * The cohort manager is always added to the list.
```

Comment:
Note: this is not a change in behavior.  In the existing code, exceptions were being caught and logged (and not rethrown) in a catch(Throwable) block.  I'm just fixing the comment to reflect what is (and was) actually happening.  I think having it in the list make sense to that the apis to get the cohort status will show cohorts that are still in the connecting/retrying/etc state.